### DISCUSSION TITLE
Release of aiohttp version 3.8

### DISCUSSION THREAD

**Discussion by maikelvdh** [aio-libs/aiohttp#5950] [Q&A] [answered]
Is there any timeline when 3.8 version will be released? Looking forward to see https://github.com/aio-libs/aiohttp/issues/4431 being part of a new released version to allow full usage of proxy setup.

**Comment by asvetlov** [comment]
Soon, in a few days. Need to test autobuild scripts and maybe merge a couple of pull requests.

**Comment by webknjaz** [comment, marked as answer]
aiohttp v3.8.0 has been released last night: https://pypi.org/project/aiohttp/3.8.0.
